apps/src/sites/code.org/pages/public/teacher-dashboard/sections.js

```diff
@@ -59,3 +60,33 @@ export function unmountSyncOauthSectionControl() {
 function syncOauthSectionMountPoint() {
   return document.getElementById('react-sync-oauth-section');
 }
+
+/**
+ * Render the login type details and controls for changing login type
+ * at the bottom of the manage students tab.
+ * @param {number} sectionId
+ */
+export function renderLoginTypeControls(sectionId) {
+  registerReducers({teacherSections});
+  const store = getStore();
+
+  store.dispatch(asyncLoadSectionData());
```

Comment:
Yeah, a smarter behavior around this async loading would be really valuable.  I'd like to suggest we do that in a followup PR because I actually tried it and it's fairly involved - I'd like to move toward synchronous loading and skip the async step when possible, but also support manual invalidation of the local section content to support things like [the workaround to rerender the React component correctly when the angular state changes](https://github.com/code-dot-org/code-dot-org/pull/16989/commits/1f871e2f47f6175d6ca637d8848465cf03a53f77).